mappings/net/minecraft/client/gui/widget/layout/RotatableLayoutWidget.mapping

```diff
@@ -0,0 +1,86 @@
+CLASS net/minecraft/unmapped/C_zuxsitrm net/minecraft/client/gui/widget/layout/RotatableLayoutWidget
+	FIELD f_ochhixwu children Ljava/util/List;
+	FIELD f_wnzzziql orientation Lnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;
+	FIELD f_xpienxli layoutSettings Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD <init> (IIIILnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 height
+	METHOD <init> (IILnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;)V
+		ARG 1 width
+		ARG 2 height
+		ARG 3 orientation
+	METHOD m_hjuyuxiv (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)V
+		ARG 1 child
+	METHOD m_nrxugcux add (Lnet/minecraft/unmapped/C_dpukplka;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+	METHOD m_rsgnlyvt add (Lnet/minecraft/unmapped/C_dpukplka;Lnet/minecraft/unmapped/C_pjarmpvz;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+		ARG 2 layoutSettings
+	METHOD m_syxvupud getLayoutSettings ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_wauzovfj copyLayoutSettings ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_ywlwqxpm add (Lnet/minecraft/unmapped/C_dpukplka;Ljava/util/function/Consumer;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+		ARG 2 layoutSettingsConsumer
+	CLASS C_ekgfxrpf Orientation
+		METHOD m_fnqdcoug setRotatedY (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;II)V
+			COMMENT Sets the "y" coordinate of the given {@code widget} rotated with this orientation.
```

Comment:
```suggestion
			COMMENT Sets the Y coordinate of the given {@code widget} rotated with this orientation.
```

Reply:
No no no! Coordinates are **always** lowercase in math notation.

Reply:
Unless you have a very good reason to make it capital, it should be lowercase.

Reply:
I'm confused; i thought this was a standard on our javadocs
And yes, math does have the fancy `x` and such but we don't have TeX on our javadoc and uh, just having a lowercase letter doesn't make sense here; i made this request because it felt like it blended with the rest of the text, which is bad; it didn't feel distinct, and considering that stuff like Blender does use capital letters for coordinates? I feel like my gut feelings are justified here
![Screenshot of Blender showing the transform fields of the default cube, with the usage of capital letters for coordinates being evident](https://github.com/QuiltMC/quilt-mappings/assets/85590273/8b2d3559-d6f9-49b3-a052-e3d127492617)
So, if we want to follow a lowercase standard, we ought to use `{@code x}` in order to make it distinct *and* we must make it a formal standard; else? It has to be capital

Reply:
I'd prefer flat `x` over `{@code x}`, since we usually use code for parameters it may get people to do a double take

Reply:
capitals it is then; i don't believe quotes are a solution

Reply:
mmm no I meant "the x coordinate", not "the X coordinate", "the {@code x} coordinate", or "the "x" coordinate"

Reply:
1. I think it should be lowercase because that's how it's used in math
2. I agree with EnnuiL that "it felt like it blended with the rest of the text, which is bad"
3. I don't think we should make it distinct with `{@code}`, because it's not code, that's misleading
So, how about it's made distinct with other text formatting, like some combination of bold/italic/`<strong>`/etc?
I'm partial to italics, but idk if that's enough to make it stand out in all cases.

Reply:
I support italics, I don't think it needs to be super flashy

Reply:
Italics is the closest way of making the coordinate letters look like math notation without making Java implement TeX on their javadoc engine :p

Reply:
It uses quotes because it's not necessarily the actual y coordinate by the way.

Reply:
I definitely _wouldnt_ capitalize it, whatever you pick. The issue isn't so much that coords are generally lowercase in math - it's that uppercase letters have several other connotations! I see an X somewhere and my brain goes "matrix!"